src/debug/debug.scss

```diff
@@ -85,4 +85,9 @@
   span{
     float:right;
   }
+
+  .ffwdme-debug-wrapper>button {
```

Comment:
You are already in the `ffwdme-debug-wrapper` scope. `>button` should be sufficient here.

Reply:
Corrected.